_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+```
+
+## If you are using the IAM instance roles
+
+```bash
+# Update these with the information from the bucket that you created
+export BUCKET_NAME=spinnaker-abcxyz
+export REGION=us-west-2
+
+# This will prompt for the secret key
+hal config storage s3 edit \
+    --bucket ${BUCKET_NAME} \
+    --region ${REGION}
+
+hal config storage edit --type s3
+```
+
+## If you want to use a specific folder in the bucket
+
+By default, Halyard will configure Spinnaker to use the folder `front50` in your S3 bucket. You can configure it to use a different folder with this command:
+
+```bash
+ROOT_FOLDER=not_front50
+hal config storage s3 edit --root-folder ${ROOT_FOLDER}
+```
+
+# Set up Gate to listen on the `/api/v1` path
+
+Create these files:
+
+`/home/spinnaker/.hal/default/profiles/gate-local.yml`
+```yml
+server:
+  servlet:
+    context-path: /api/v1
+```
+
+`/home/spinnaker/.hal/default/service-settings/gate.yml`
+```yml
+healthEndpoint: /api/v1/health
+```
+
+You can run this to automatically create these two files:
+
+```bash
+mkdir -p /home/spinnaker/.hal/default/{profiles,service-settings}
+
+tee /home/spinnaker/.hal/default/profiles/gate-local.yml <<-'EOF'
+server:
+  servlet:
+    context-path: /api/v1
+EOF
+
+tee /home/spinnaker/.hal/default/service-settings/gate.yml <<-'EOF'
+healthEndpoint: /api/v1/health
+EOF
+```
+
+# Choose the Spinnaker version
+
+Before Halyard will install Spinnaker, you should specify the version of Spinnaker you want to use.
+
+You can get a list of available versions of spinnaker with this command:
+
+```bash
+hal version list
+```
+
+_If you are installing Armory Spinnaker, you will get a version that starts with `2.x.x`_
+
+_If you are installing OSS Spinnaker and using `gcr.io/spinnaker-marketplace/halyard:stable`, you will get a version that starts with `1.x.x`_
+
+And then you can select the version with this:
+
+```bash
+# Replace with version of choice:
+export VERSION=$(hal version latest -q)
+echo ${VERSION}
+hal config version edit --version $VERSION
+```
+
+# Install Spinnaker
+
+Now that your halconfig is completely configured for the initial Spinnaker, you can tell Halyard to actually install Spinnaker:
+
+```bash
+hal deploy apply --wait-for-completion
+```
+
+Once this is complete, congratulations! Spinnaker is installed. Now we have to access and expose it.
+
+## Connect to Spinnaker using `kubectl port-forward`
+
+If you have kubectl on a local machine with access to your Kubernetes cluster, you can test the status of your Spinnaker instance by doing a port-forward:
+
+First, tell Spinnaker about its local endpoint for localhost:9000/api/v1:
+
+```bash
+hal config security api edit --override-base-url http://localhost:9000/api/v1
+
+hal deploy apply --wait-for-completion
+```
+
+Wait for the pods to stand up.  Then set up two port forwards:
+
+```bash
+NAMESPACE=spinnaker
+kubectl -n ${NAMESPACE} port-forward svc/spin-deck 9000 &
+kubectl -n ${NAMESPACE} port-forward svc/spin-gate 8084 &
```

Comment:
may need to indicate the usage of the `kubeconfig-aws`

Reply:
removed references to external kubeconfig